EnglishTranslation-master/QUEST.tsv

```diff
@@ -3076,31 +3076,31 @@ QUEST_20151224_003083	For example if you find the legendary fantastic library, h
 QUEST_20151224_003093	All beings in this world are connected in some way.{nl}We can end all conflicts if we accept each other as one and the same.
 QUEST_20151224_003094	Shapeshifting is a very profound type of magic, but simply learning it will not make you stronger.{nl}If you can't use it in battle, you're better off learning something else.
 QUEST_20151224_003095	For Thaumaturges, it's the battle that matters.{nl}You need to control others' properties and shapes in a way that's favorable to you.
-QUEST_20151224_003096	Orsha is also being plagued with all sorts of accidents, so nothing works out.{nl}What we really need now are talented strong people who can help us.
+QUEST_20151224_003096	Orsha is also being plagued with all sorts of incidents, so nothing's been going as planned.{nl}What we really need now are strong and talented people who can help us.
 QUEST_20151224_003097	We don't focus on a single property.{nl}Us Elementalists combine various different elemental properties in an efficient way.
 QUEST_20151224_003098	The advantage of using a bow is that you can finish an opponent from afar.{nl}Of course that is meaningless if you can't aim, haha.
 QUEST_20151224_003099	Archers are already invincible to any opponent out there, {nl}but there is one type that actually makes them stronger.{nl}I'm talking about the flying-type enemies with cloth armor.
-QUEST_20151224_003100	If you run away when there's too many enemies, look somewhere else.{nl}I'm not into defeating only one at a time.
-QUEST_20151224_003101	A Rangers' techniques are specialized to handle multiple opponents at once.{nl}If you're going to defend this many settlers, isn't that an advantage for you?
+QUEST_20151224_003100	If you run away when there's too many enemies, look somewhere else.{nl}I'm not into defeating only one enemy at a time.
+QUEST_20151224_003101	Rangers' techniques are specialized in handling multiple opponents at once.{nl}If you're going to defend this many settlers, wouldn't these techniques be an advantage for you?
 QUEST_20151224_003102	Shields, rocks, monsters, you need to use anything you can.{nl}There's no such thing as being shy in battle.
 QUEST_20151224_003103	Of course flashy techniques are good too...{nl}When you're not hurt and in no danger to die, that is.
 QUEST_20151224_003104	Using traps to affect many enemies at once is the specialty of Sappers.{nl}We use techniques adopted from military engineering.
 QUEST_20151224_003105	I believe a Sapper always needs to be cautious.{nl}Any trap you set up can completely change the outcome of the battle.
-QUEST_20151224_003106	You can't think of Hunters without companions.{nl}They aren't just comrades in battle, but also trustworthy friends.
+QUEST_20151224_003106	Hunters aren't Hunters without their companions.{nl}Companions aren't just comrades in batte, they are trustworthy friends.
```

Comment:
"comrades in battle" or "comrades in arms" are both phrases that work... do you have a suggestion on how else the line could be worded or were you just confused at the phrase?